ignite/contrib/handlers/param_scheduler.py

```diff
@@ -8,28 +8,38 @@ class ParamScheduler(object):
     training.
 
     Args:
-        optimizer (`torch.optim.Optimizer`): the optimizer to use
+        optimizer (`torch.optim.Optimizer`/dict): the optimizer or parameters group to use
         param_name (str): name of optimizer's parameter to update
         save_history (bool, optional): whether to log the parameter values
             (default=False)
     """
     def __init__(self, optimizer, param_name, save_history=False):
+
         self.optimizer = optimizer
```

Comment:
`self.optimizer` becomes unused ?

Reply:
It is used in ConcatScheduler

Reply:
@amitibo Okay, I see the dependency, we need to setup the optimizer inside ConcatScheduler. But we can then set `self.optimizer = optimizer` in ConcatScheduler only.